Sentinel/Classes/Core/UIImage+Bundle.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  UIImage+Bundle.swift
+//  Sentinel
+//
+//  Created by Zvonimir Medak on 30.09.2021..
+//
+
+import Foundation
+extension UIImage {
+
+    static func loadImageFromBundle(name: String) -> UIImage? {
```

Comment:
This could be just load(fromBundle name: String) or load(fromBundleNamed name: String)